### PR TITLE
Merge rc v1.6.0 feat pubkey conv 2023.03.10

### PR DESCRIPTION
**Pull request by iulianpascalau**
## Reasoning behind the pull request
- 
- 
- 
  
## Proposed changes
- 
- 
- 

## Testing procedure
- 
- 
- 

## Pre-requisites

Based on the [Contributing Guidelines](https://github.com/multiversx/mx-chain-go/blob/master/.github/CONTRIBUTING.md#branches-management) the PR author and the reviewers must check the following requirements are met:
- was the PR targeted to the correct branch?
- if this is a larger feature that probably needs more than one PR, is there a `feat` branch created?
- if this is a `feat` branch merging, do all satellite projects have a proper tag inside `go.mod`?

### CODE REVIEW

**Review by bogdan-rosianu — APPROVED** [review]

File: common/enablers/enableEpochsHandler.go [hunk 1]
```diff
@@ -33,96 +33,96 @@ func NewEnableEpochsHandler(enableEpochsConfig config.EnableEpochs, epochNotifie
...
+	handler.setFlagValue(epoch >= handler.enableEpochsConfig.BalanceWaitingListsEnableEpoch, handler.balanceWaitingListsFlag, "balanceWaitingListsFlag", epoch, handler.enableEpochsConfig.BalanceWaitingListsEnableEpoch)
+	handler.setFlagValue(epoch > handler.enableEpochsConfig.ReturnDataToLastTransferEnableEpoch, handler.returnDataToLastTransferFlag, "returnDataToLastTransferFlag", epoch, handler.enableEpochsConfig.ReturnDataToLastTransferEnableEpoch)
+	handler.setFlagValue(epoch >= handler.enableEpochsConfig.SenderInOutTransferEnableEpoch, handler.senderInOutTransferFlag, "senderInOutTransferFlag", epoch, handler.enableEpochsConfig.SenderInOutTransferEnableEpoch)
+	handler.setFlagValue(epoch >= handler.enableEpochsConfig.StakeEnableEpoch, handler.stakeFlag, "stakeFlag", epoch, handler.enableEpochsConfig.StakeEnableEpoch)
+	handler.setFlagValue(epoch >= handler.enableEpochsConfig.StakingV2EnableEpoch, handler.stakingV2Flag, "stakingV2Flag", epoch, handler.enableEpochsConfig.StakingV2EnableEpoch)
+	handler.setFlagValue(epoch == handler.enableEpochsConfig.StakingV2EnableEpoch, handler.stakingV2OwnerFlag, "stakingV2OwnerFlag", epoch, handler.enableEpochsConfig.StakingV2EnableEpoch)
```
[the remark this answers is not in the document]
Comment (iulianpascalau) [reply common/enablers/enableEpochsHandler.go:54]: No, can be seen here:
https://github.com/multiversx/mx-chain-go/blob/1e52082ecfa4bc0a16aff5d04ece1d92834dff59/common/enablers/enableEpochsHandler.go#L54

File: common/enablers/enableEpochsHandler.go [hunk 2]
```diff
@@ -33,96 +33,96 @@ func NewEnableEpochsHandler(enableEpochsConfig config.EnableEpochs, epochNotifie
...
+	handler.setFlagValue(epoch > handler.enableEpochsConfig.StakingV2EnableEpoch, handler.stakingV2GreaterEpochFlag, "stakingV2GreaterEpochFlag", epoch, handler.enableEpochsConfig.StakingV2EnableEpoch)
```
[the remark this answers is not in the document]
Comment (iulianpascalau) [reply common/enablers/enableEpochsHandler.go:55]: No, can be seen here on rc/v1.6.0:
https://github.com/multiversx/mx-chain-go/blob/1e52082ecfa4bc0a16aff5d04ece1d92834dff59/common/enablers/enableEpochsHandler.go#L55

**Review by schimih — APPROVED** [review]

**Review by SebastianMarian — APPROVED** [review]

File: common/enablers/enableEpochsHandler.go (hunk 1 above)
[the remark this answers is not in the document]
Comment (SebastianMarian) [reply common/enablers/enableEpochsHandler.go:54]: ok

File: common/enablers/enableEpochsHandler.go (hunk 2 above)
[the remark this answers is not in the document]
Comment (SebastianMarian) [reply common/enablers/enableEpochsHandler.go:55]: ok

**Pull request merged**